### ISSUE TITLE
Error code for bare except

### ISSUE THREAD

**Issue by bsipocz** [PyCQA/pycodestyle#579]
I didn't see any error/warning code for bare except usages, however PEP8 is rather clear about them. Would it be possible to add one? E.g. it would help us to keep e.g. astropy clean of these, https://github.com/astropy/astropy/pull/5335#issuecomment-247760968.

```


When catching exceptions, mention specific exceptions whenever possible instead of using a bare except: clause.

For example, use:

try:
    import platform_specific_module
except ImportError:
    platform_specific_module = None

A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException: ).

A good rule of thumb is to limit use of bare 'except' clauses to two cases:

   * If the exception handler will be printing out or logging the traceback; at least the user will be aware that an error has occurred.
   * If the code needs to do some cleanup work, but then lets the exception propagate upwards with raise . try...finally can be a better way to handle this case.

```

**Comment by sigmavirus24** [comment]
It should be possible to add one easily. I think flake8-bugbear checks for these. So if you're using Flake8 already, you can add that to your list of requirements today and get that feature now.

**Comment by bsipocz** [comment]
The plan would be to enforce it by adding it to our style checker travis run, so ultimately it would be great to have it in a command line tool (=pycodestyle :) ).

**Comment by sigmavirus24** [comment]
@bsipocz Flake8 is a command line tool. It already uses pycodestyle for style checks. Adding flake8-bugbear would just add the extra checks you're looking for. I do not understand your comment :(.

**Comment by bsipocz** [comment]
Oh, ok, now I see. Somehow I assumed it's an editor plugin.

**Comment by sigmavirus24** [comment]
Fixed in #592. Thanks for opening this @bsipocz!
